### PR TITLE
Add collection and channel query param support to Twitch

### PR DESCRIPTION
**Pull request by zslabs**
👋 This PR adds two additional params to the embed that are available for iFrame - `collection` and `channel`

### CODE REVIEW

File: @pauliescanlon/gatsby-mdx-embed/src/components/Twitch/Twitch.tsx [hunk 1]
```diff
@@ -32,7 +38,7 @@ export const Twitch: FunctionComponent<ITwitchProps> = ({
     >
       <iframe
         title={`twitch-${twitchId}`}
-        src={`https://player.twitch.tv/?autoplay=${autoPlay}&t=${h}h${m}m${s}s&video=v${twitchId}`}
+        src={`https://player.twitch.tv/?autoplay=${autoPlay}&t=${h}h${m}m${s}s&video=v${twitchId}&channel=${channel}&collection=${collection}`}
```
Comment (PaulieScanlon) [inline @pauliescanlon/gatsby-mdx-embed/src/components/Twitch/Twitch.tsx]: Hey @zslabs thanks for the contribution, Is it possible to add an example story in the mdx so show what the `twitchId` url might look like if  a channel or collection was included

Comment (zslabs) [reply @pauliescanlon/gatsby-mdx-embed/src/components/Twitch/Twitch.tsx]: 👋 I've updated the examples (and added a "channel" one) to utilize the required `parent` param that needs to be passed through to each instance. If you run this locally, you'll need to change these examples to `localhost` to be able to see them. I honestly couldn't find a collection 🤷 but if you have an example one I'm happy to update the PR to include it.

File: docs/src/components/twitch.stories.mdx [hunk 2]
```diff
@@ -13,11 +13,21 @@ Display a Twitch video by including the component in your `.mdx`. The `twitchId`
 ## Basic usage
 
 <Preview>
-  <Twitch twitchId="612049948" />
+  <Twitch twitchId="679474192" parent="gatsby-mdx-embed.netlify.app" />
```
Comment (PaulieScanlon) [inline docs/src/components/twitch.stories.mdx:16]: @zslabs Thanks for this! Is this why in the [docs](https://gatsby-mdx-embed.netlify.app/?path=/docs/components-twitch--page) the Twitch videos stopped working? is it a new Twitch thing?

**Review by PaulieScanlon — APPROVED** [review]

File: docs/src/components/twitch.stories.mdx (hunk 2 above)
Comment (zslabs) [reply docs/src/components/twitch.stories.mdx:16]: Yep that's why! It looks like it was changed in Feb of this year https://discuss.dev.twitch.tv/t/twitch-embedded-player-updates-in-2020/23956

Comment (PaulieScanlon) [reply docs/src/components/twitch.stories.mdx:16]: Ah ha! Brilliant thanks for sorting it. I'll get it merged and do a release in the morning. Great stuff. Thanks again!

**Pull request merged**